02-FlexBox/assignment/css/style.scss

```diff
@@ -1,6 +1,98 @@
-// Import partials here
-@import 'base.normalize';
+// Imported partials
 @import 'settings.variables';
 @import 'settings.responsive';
 @import 'components.content';
 
+/* Basic resets. Didn't @import base.normalize because it caused a problem I couldnt fix */
```

Comment:
Not a problem. Maybe during office hours you could show me what happened and we can fix it, but no biggie.